server/src/routes/api/v1/shoeReviewsRouter.js

```diff
@@ -23,4 +23,26 @@ shoeReviewsRouter.post("/", async (req, res) => {
   }
 })
 
+shoeReviewsRouter.patch("/", async (req, res) => {
+  const { type, id } = req.body
+  try {
+    let review
+    if(type === "upvote"){
+      review = await Review.query().patchAndFetchById(id, {votes: raw('votes + 1')})
+    }
+    if(type === "downvote"){
+      review = await Review.query().patchAndFetchById(id, {votes: raw('votes - 1')})
+    }
+    if(!review) {
+      res.status(500).json({ errors: review })
```

Comment:
I see what you're doing here, but we will want to separate votes from the review